.circleci/config.yml

```diff
@@ -1,14 +1,27 @@
 version: 2.1
 
-jobs:
-  hello:
+executors:
+  ruby:
     docker:
-      - image: alpine
+      - image: circleci/ruby:2.7.1
+
+jobs:
+  test:
+    executor: ruby
     steps:
-      - run: echo "Hello, world! 👋"
+      - checkout
```

Comment:
Well... I guess this is a good example of "show the history of your project since the very beginning" ;-)